package.json

```diff
@@ -8,7 +8,8 @@
     "audit:ci": "audit-ci --low",
     "bbcA11y": "wait-on -t 20000 http://localhost.bbc.com:7080/news/articles/c6v11qzyv8po && bbc-a11y",
     "build:ci": "export NODE_ENV=production && rm -rf build && npm run build:test && npm run build:live",
-    "build": "rm -rf build && cp envConfig/local.env .env && npm run webpack:prod && npm run test:size",
+    "build:dependency": "npm explore 'packages/buildDependencies'",
```

Comment:
I think this should be prebuilt, also in the explore script you need to refer to a script inside that packages package.json